sympy/logic/tests/test_inference.py

```diff
@@ -313,3 +313,15 @@ def test_satisfiable_all_models():
     result = satisfiable(Or(*X), all_models=True)
     for i in range(10):
         assert next(result)
+
+def test_z3():
```

Comment:
This should be skipped when z3 is not installed. See sympy/utilities/tests/test_lambdify.py for an example of how to do this.